.cirrus.yml

```
@@ -14,7 +14,7 @@ task:
     - pkg install -y autoconf bison gmake re2c icu libiconv png freetype2 enchant2 bzip2 krb5 t1lib gmp tidyp libsodium libzip libxml2 libxslt openssl oniguruma pkgconf webp
```

```suggestion
    - pkg install -y autoconf bison gmake re2c icu libiconv png freetype2 enchant2 bzip2 krb5 t1lib gmp tidyp libsodium libzip libxml2 libxslt openssl oniguruma pkgconf webp libavif
```
Should fix the CI failure.

---

Argh. I [had added that](https://github.com/php/php-src/commit/76f3a98dcb93a4906e3021192e42b4cdf5dd6d03#diff-62587956f943bb2503db7bc6dd27d0d888074a1c0ecaab3f570ad611aff0f7bb), but it got lost when I merged. Thanks for noticing!